### PR TITLE
Implemented Drag and Drop

### PR DESCRIPTION
**Pull request by gestureleft**
This first iteration supports:
- animated dragging of elements across the screen
- having multiple drop targets and multiple draggables
- passing data between what is being dragged and the drop target it is being dropped on
- animating back to original position if drop target is hit

### CODE REVIEW

File: packages/TorneloScoresheet/src/types/ChessBoardPositions.ts [hunk 1]
```diff
@@ -31,6 +31,74 @@ export type Row = '1' | '2' | '3' | '4' | '5' | '6' | '7' | '8';
 export type Column = 'a' | 'b' | 'c' | 'd' | 'e' | 'f' | 'g' | 'h';
 export type Position = `${Column}${Row}`;
 
+// All the squares on a board
+export const BOARD_SQUARES: Position[] = [
```
Comment (nocommentcode) [inline packages/TorneloScoresheet/src/types/ChessBoardPositions.ts:35]: what's this for?

**Review by nocommentcode — APPROVED** [review]
looks good :)

File: packages/TorneloScoresheet/src/types/ChessBoardPositions.ts (hunk 1 above)
Comment (gestureleft) [reply packages/TorneloScoresheet/src/types/ChessBoardPositions.ts:35]: Ah yep, this doesn't belong in this PR. Thanks!

**Review by nocommentcode — APPROVED** [review]
lgtm

**Pull request merged**